src/needsahome.jl

```diff
@@ -99,11 +99,19 @@ Display the content of `VariableNodeData` to console for a given factor graph an
 Dev Notes
 - TODO split as two show macros between AMP and DFG
 """
-printVariable(fgl::AbstractDFG, vsym::Symbol, solveKey::Symbol=:default) = print(getVariable(dfg, sym))
+printVariable(dfg::AbstractDFG, sym::Symbol, solveKey::Symbol=:default) = print(getVariable(dfg, sym))
 
-print(fgl::AbstractDFG, fsym::Symbol) = isVariable(dfg,fsym) ? printVariable(dfg, fsym) : printFactor(dfg, sym)
+print(dfg::AbstractDFG, sym::Symbol) = isVariable(dfg,sym) ? printVariable(dfg, sym) : printFactor(dfg, sym)
 
 
+#NOT too many aliases on PPE, brought back from deprecated DF
+
+getVariablePPEs(v::VariableDataLevel1) = getPPEDict(v)
+
+getPPEs(vari::VariableDataLevel1) = getPPEDict(v)
```

Comment:
This was removed because its inconsistent behavior in the api. see getVariables, getFactors, etc.
You have to keep the other drivers in mind. Its not recommended to work with the dict directly (its there for more advanced use) and changes should be made through the api. 

To fit in with the API `getPPEs` should return a vector of all the PPE in the variable, thats why the deprecation, to change in the next cycle. The names still have a bit of a problem with just using a plural (as with getSolverData?), we can maybe consider using getSolverDataAll, getPPEAll etc.